rest-api/pom.xml

```diff
@@ -33,7 +33,7 @@
     </parent>
 
     <artifactId>pipeline-rest-api</artifactId>
-    <version>2.2-SNAPSHOT</version>
+    <version>2.2-optimized-SNAPSHOT</version>
```

Comment:
Will be reverted before merge, in this case it's to guarantee artifact for benchmarking.